api-catalog-services/src/main/java/com/ca/mfaas/apicatalog/services/cached/CachedProductFamilyService.java

```diff
@@ -236,16 +255,15 @@ private void checkIfContainerShouldBeUpdatedFromInstance(InstanceInfo instanceIn
      * @param instanceInfo the service instance
      * @return a APIService object
      */
-    private APIService createAPIServiceFromInstance(InstanceInfo instanceInfo) {
+    public APIService createAPIServiceFromInstance(InstanceInfo instanceInfo) {
```

Comment:
Why is this method public? Because of tests?

Reply:
I think I had changed it for that purpose but eventually is not used so I can set it back to private